app/views/layout.html

```diff
@@ -18,4 +18,5 @@
 
 {% block body_end %}
   {% include "includes/scripts.html" %}
+  <!-- GOV.UK Prototype kit {{releaseVersion}} -->
```

Comment:
I'd like to put something right at the top, but the template doesn't currently provide the right hooks.

At the bottom is quite common for org signatures / notices, so I think it can go here too?